src/aap_eda/api/serializers/event_stream.py

```diff
@@ -43,6 +43,23 @@ class EventStreamInSerializer(serializers.ModelSerializer):
         error_messages={"null": "EdaCredential is needed"},
     )
     uuid = serializers.UUIDField(required=False, allow_null=True)
+    test_mode = serializers.BooleanField(
+        required=False,
+        help_text=(
```

Comment:
Is there a plan to obsolete test_mode in the future? If so we may reflect it in the help_text.
Can we accept both test_mode and forward_events as long as they are not contradict each other?